tests/PerformanceTests/Microsoft.Identity.Web.Perf.Client/appsettings.json

```diff
@@ -0,0 +1,11 @@
+{
+    "IntegrationTestServicesBaseUri": "https://msidwebintegrationtestservice.azurewebsites.net",
+    "TenantDomain": "msidwebperformancetesting.onmicrosoft.com",
+    "DurationInMinutes": "1440",
+    "UserPassword": "secret_here",
+    "TestUri": "/SecurePage/GetTokenForUserAsync",
+    "ApiScopes": "api://b05c7590-7c68-4aba-9454-38b1f787a989/.default",
+    "ClientId": "91f7fafd-e19a-4637-b3b9-1550fb132469",
+    "UsersStartIndex": "1", //start w/at least one
+    "UsersEndIndex": "10000"
+}
```

Comment:
nit: maybe add a note that we only have 10k users. :)

Reply:
It"s a console app.
If it gets its creds from KeyVault, you can run it with a release definition (used for tests), triggered by the CI Build